### ISSUE TITLE
Add an option to send a link to the profession itself

### ISSUE THREAD

**Issue by stevin05** [stevin05/CraftScan#14]

**Comment by roband7** [comment]
Yes, this would be a great little addition. Now it always goes like:

player in /2: LF tailor
addon replies: I can do Tailoring
player in /w: Can you please link?

If the Tailoring profession link was included in the first reply instead ot the text Tailoring this would be much smoother.

**Comment by stevin05** [comment]
After a bit of investigation, generating the link requires you to be on the character with the profession (seems obvious now). It also requires you to open the profession window to generate the link (which can be automated, but still has to open that window).

My profession window is insanely laggy due to CraftSim (probably need to go hit some cleanup button, but it's likely a common issue among crafters), so the idea of clicking the reply button only for the screen to freeze for a few seconds while it opens the profession window to generate the link seems like a terrible experience.

Going to say this one is not worth it. I'm rarely asked for the profession link anyway.
